### PR TITLE
Added Haskell support to NormEZ

### PR DESCRIPTION
**Pull request by jackgdll**
I added most of the Haskell coding style rules to NormEZ and refactored some of the code according to the rubocop linter

### CODE REVIEW

**Comment by jackgdll** [comment]
I also added  a makefile to install and uninstall normez

File: README.md [hunk 1]
```diff
@@ -144,5 +166,6 @@ You want to add awesome features to NormEZ? Here's how:
 [Here](https://github.com/ronanboiteau/NormEZ/graphs/contributors) is the list of NormEZ's contributors. Thanks to everyone who helped developing this project!
 
 <!-- Links -->
-[Epitech coding-style]: /epitech_c_coding_style.pdf
+[Epitech C coding-style]: /epitech_c_coding_style.pdf
+[Epitech Haskell coding-style]: /epitech_c_coding_style.pdf
```
Comment (ronanboiteau) [inline README.md]: Wrong link:

```suggestion
[Epitech Haskell coding-style]: /epitech_haskell_coding_style.pdf
```

File: README.md [hunk 2]
```diff
@@ -114,12 +129,19 @@ Here are the [Epitech coding-style] violations ***NOT YET*** checked by NormEZ.
...
+<img alt="Major infraction" src="/artwork/direction_arrow_red_up.png" width="12" height="12"/> Forbidden language extentions.<br/>
+<img alt="Major infraction" src="/artwork/direction_arrow_red_up.png" width="12" height="12"/> Contents of delivery folder<br/>
+
 ## Bugs
 
 ### Known issues
 
  - NormEZ doesn't make the difference between strings/comments & code. Examples: a commented forbidden function will be flagged, as well as a commented `;` (multiple assignments on the same line), etc.
  - The check for functions containing more than 20 lines doesn't work yet with the new coding style v3.1. [See related issue.](https://github.com/ronanboiteau/NormEZ/issues/20)
+ - The check of guards that should be expressed as pattern matching is not very accurate, it flags guards that use only direct comparisons (or `==`)
```
Comment (ronanboiteau) [inline README.md]:
```suggestion
 - For Haskell, the check of guards that should be expressed as pattern matching is not very accurate, it flags guards that use only direct comparisons (or `==`)
```

**Review by ronanboiteau — APPROVED** [review]
Great additions! Thank you 😃

**Comment by ronanboiteau** [comment]
Released in v2.0.0: https://github.com/ronanboiteau/NormEZ/commit/f6bfd0af700adb97e599beb5e2d5a3aa80f5a2d0

**Pull request merged**